Tasks/KubernetesManifestV0/task.json

```diff
@@ -13,12 +13,13 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 166,
-        "Patch": 1
+        "Minor": 167,
+        "Patch": 0
```

Comment:
This task has helm install logic too! Validate it here as well

Reply:
This task uses helmUtility from kubernetes-common-v2. That's already been updated.